### PR TITLE
Toaster notifications and relaxed URL/email validation

### PR DESCRIPTION
**Pull request by AdamDanielKing**
Changes:
- Adds Toaster and ToasterContainer for showing notifications.
- Relaxes URL and email validation.
- Adds SCSS and commands for compiling SCSS and JS.
- Adds eslint-plugin-react.
- Moves from babel-preset-es2015 to babel-preset-env.
- Adds licence to package.json.
- Adds to README.md slightly.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  extends: ['plugin:polymath/recommended']
-}
\ No newline at end of file
+  "extends": [
+    "plugin:react/recommended",
```
Comment (bshevchenko) [inline .eslintrc:3]: When you need react eslint rules use eslint-plugin-polymath-react

File: src/components/toaster.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import PropTypes from 'prop-types'
+
+import styles from '../style.css'
+
+export class Toaster extends Component {
+  state = {
+    toasts: []
+  }
+
+  show (toast, duration = 4000) {
```
Comment (bshevchenko) [inline src/components/toaster.js]: Don't pack arguments into the one Object variable. It makes interface unclear and requires additional types specification from each user (see https://github.com/PolymathNetwork/polymath-issuer/pull/1/files#diff-ac3c50ae15a631981374f09a64815339R34).
Instead we can use extra class like – https://github.com/PolymathNetwork/polymath-issuer/commit/ea6d7f1d496476be4598437266976e7cd2249e51#diff-96547c5732d1f2950d1f0f434eeb6796R20 – or just spread and specify Flow type for each of them.

File: src/components/toaster.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+    return (
+      <div className="pui-toaster">
+        {toastElements}
+      </div>
+    )
+  }
+}
+
+export const ToasterContainer = ({ children }) => (
```
Comment (bshevchenko) [inline src/components/toaster.js]: Why do we need this extra component?

**Review by bshevchenko — CHANGES_REQUESTED** [review]

File: src/components/toaster.js (hunk 3 above)
Comment (AdamDanielKing) [reply src/components/toaster.js]: The only reason for that container is to contain the toaster in something with the pui-toaster-container class. We could export the class name instead. What do you suggest?

Comment (bshevchenko) [reply src/components/toaster.js]: See review https://github.com/PolymathNetwork/polymath-issuer/pull/1

File: flow-typed/npm/style-loader_vx.x.x.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+// flow-typed signature: f47a71cb085178a55158a08fcac0a670
```
Comment (bshevchenko) [inline flow-typed/npm/style-loader_vx.x.x.js:1]: Are you sure that all these flow-typed/npm files should be placed in the repository?

File: src/components/toaster.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
+// @flow
```
Comment (bshevchenko) [inline src/components/toaster.js:1]: I think we can omit this tag everywhere

**Review by bshevchenko — CHANGES_REQUESTED** [review]

**Pull request merged**